### DISCUSSION TITLE
some missing MPP packages

### DISCUSSION THREAD

**Discussion by squassina** [7Ji/orangepi5-archlinuxarm#40] [Q&A]
Hey, again, thanks for the great image for OPI5+!

I noticed that two packages are missing from the repo: ffmpeg-mpp (that I was able to install using the -git version) and chromium-mpp.

Are these still to be available? Or are there other options to have the mpp version of chromium that I could run?

Thank you!

**Comment by 7Ji** [comment]
For ffmpeg, use -git, the none-git version was deprecated 

For chromium, the previously 114 was broken and it can't be built against the current ALARM dependencies, I'm trying to get 122 built and worked through some build issues but still not having finished it (breaks at 51000/59000), so it's currently not possible to either build or install chromium-mpp

**Reply by squassina** [reply]
oh, that's a pity! Is it possible to change the script that upload your packages to replace only the new ones without deleting the ones that are not being uploaded? That way, we could use the ones available while newer ones are being built.

Thanks!

**Reply by 7Ji** [reply]
> That way, we could use the ones available while newer ones are being built

You can't, the builder would only do a full update deleting old packages after a complete successful repo build. If not, it only does partial update. 114 was deleted on purpose by myself as it's broken and can't be used on latest ALARM. You can't lock the dependencies and call it a day, as Arch is a rollong release.
